GiniVision/Classes/Core/Screens/Camera/CameraButtonsViewController.swift

```diff
@@ -184,12 +184,18 @@ final class CameraButtonsViewController: UIViewController {
         
         addImportButtonConstraints()
     }
+    
+    func enableCaptureButton(enable:Bool) {
```

Comment:
The function name is a bit misleading for me. It should be something like `toggleCaptureButtonActivation(state)`?

Reply:
you're right, I'll rename it. Thank you.